integration-templates/cal-com-v2/types.ts

```diff
@@ -0,0 +1,20 @@
+import type { EventType } from '../models';
+
+export interface EventTypeResponse {
+    status: boolean;
+    data: {
+        eventTypeGroups: EventTypeGroup[];
+    };
+}
+
+export interface EventTypeGroup {
+    teamId: string | null;
+    bookerUrl: string;
+    membershipRole: string | null;
+    profile: {
+        slug: string;
+        name: string;
+        image: string;
+    };
+    eventTypes: EventType[];
```

Comment:
not sure its advisable to mix our own models and theirs

Reply:
Will keep as is and can refine if needed later. This is for a customer, so assume they'll want a tighter model but they haven't let us know what that is yet.